PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/functions/localrun/ui/FunctionRunPanel.java

```diff
@@ -108,7 +109,7 @@ public void disposeEditor() {
 
     @Override
     protected void resetFromConfig(@NotNull FunctionRunConfiguration configuration) {
-        if (!configuration.isFirstTimeCreated()) {
+        if (MapUtils.isNotEmpty(configuration.getAppSettings())) {
```

Comment:
Will this line makes FUNCTIONS_WORKER_RUNTIME_KEY not readable from local.settings.json?

Reply:
No, `local.settings.json` will be loaded in `createUIComponents()` by `appSettingsTable.loadLocalSetting();`.

When `resetFromConfig` was invokerd by IntelliJ, if appSettings in configuration is empty, which means this is a new configuration, we will use local settings and if configuration is not empty, which means users used to save configuration before, and we will users configuration instead